onnx_xla/operator_registry.cc

```diff
@@ -0,0 +1,51 @@
+#include "onnx_xla/operator_registry.h"
+#include <iostream>
+namespace onnx_xla  {
+  
+  OperatorRegistry::OperatorRegisterOnce::OperatorRegisterOnce(const Symbol& nodeKind, TranslationFunction translator)  {
+    auto& map = OperatorRegistry::map();
+    if (map.find(nodeKind) == map.end())  {
+      map[nodeKind] = translator;
```

Comment:
this is doing two lookups, I think you can call `insert` first and assert it successfully inserted.

Reply:
This is more efficient - changed